src/templates/static/affiliate_ib_landing/type_of_partnership.jsx

```diff
@@ -260,48 +265,92 @@ const TypesOfPartnership = () => (
+                                            { text: it.L('15O') },
+                                        ],
+                                        [
+                                            { text: it.L('NEO/USD') },
+                                            { text: it.L('150') },
+                                        ],
+                                        [
+                                            { text: it.L('OMG/USD') },
+                                            { text: it.L('150') },
+                                        ],
+                                        [
+                                            { text: it.L('TRX/USD') },
+                                            { text: it.L('25') },
+                                        ],
+                                        [
+                                            { text: it.L('XLM/USD') },
+                                            { text: it.L('25') },
+                                        ],
+                                        [
+                                            { text: it.L('BTC/ETH') },
+                                            { text: it.L('20') },
+                                        ],
+                                        [
+                                            { text: it.L('BTC/LTC') },
+                                            { text: it.L('20') },
                                         ],
                                     ],
                                 }}
                             />
-
+                            <div className='affiliate-ib-calculation'>
+                                <h3 className='text-align-left'><strong>{it.L('How it’s calculated')}</strong></h3>
+                                <p className='text-align-left'>{it.L('For forex, a deal for 1 lot of EUR/USD will pay out EUR 5 in commission based on the following formula:')}</p>
+                                <img
+                                    src={it.url_for('images/pages/affiliates_ib_landing/formulas/formula-financial-forex.svg')}
+                                    width='100%'
+                                    className='gr-padding-20'
+                                />
+                                <p className='text-align-left'>{it.L('For cryptocurrency assets, a deal for 1 lot of BTC/USD (with a BTC to USD exchange rate of USD 50,000) will pay out USD 20 in commission based on the following formula:')}</p>
+                                <img
+                                    src={it.url_for('images/pages/affiliates_ib_landing/formulas/formula-financial-crypto.svg')}
+                                    width='100%'
+                                    className='gr-padding-20'
+                                />
+                                <p className='text-align-left'>{it.L('[_1]Please note[_2]:','<strong>','</strong>')}</p>
```

Comment:
We don't need to add strong tag into translation, you can just place it around it.L